### PR TITLE
ARROW-356: Add documentation about reading Parquet

### PR DESCRIPTION
**Pull request by xhochy**
Assumes #192.

### CODE REVIEW

**Comment by xhochy** [comment]
Ready for review. I've added also a bit documentation about the Pandas<->Arrow conversion. The rendered version can be found at https://pyarrow-xhochy.readthedocs.io/en/arrow-356/index.html

File: python/doc/pandas.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+| ``(U)INT{8,16,32,64}`` *with nulls* | ``float64``                                            |
++-------------------------------------+--------------------------------------------------------+
+| ``FLOAT``                           | ``float32``                                            |
++-------------------------------------+--------------------------------------------------------+
+| ``DOUBLE``                          | ``float64``                                            |
++-------------------------------------+--------------------------------------------------------+
+| ``STRING``                          | ``str``                                                |
++-------------------------------------+--------------------------------------------------------+
+| ``TIMESTAMP(unit=*)``               | ``pd.Timestamp`` (``np.datetime64[ns]``)               |
++-------------------------------------+--------------------------------------------------------+
```
Comment (wesm) [inline python/doc/pandas.rst]: Yikes, do you have a tool to do this? We could also use the csv-table directive

File: python/doc/install.rst [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+Install `pyarrow`
+~~~~~~~~~~~~~~~~~
+
+
+.. code-block:: bash
+
+    cd arrow/python
+
+    python setup.py install
```
Comment (wesm) [inline python/doc/install.rst]: Revise this to add the `python setup.py build_ext --with-parquet install`. you may also want to mention `--build-type=release`

**Review by wesm — CHANGES_REQUESTED** [review]

**Comment by wesm** [comment]
+1

**Pull request closed without merging**